### PR TITLE
Doc update - configure scopes

### LINKED ISSUE #863: [Hubs] Update "configure scopes" doc
## ⚠️ Problem
In the "manual export" section of the Configure Scopes document, the instructions currently recommend setting the directory resource ID of the scope being configured. However, this is no longer necessary.

In version 0.3 and earlier, this directory path was used to identify the scope. Starting with version 0.4, we use metadata for this purpose. Considering the new limitations around characters, we should simplify this recommendation.

## 🛠️ Solution
Improve and simplify steps.

## 🙋‍♀️ Ask for the community

We could use your help:
1. Please vote this issue up (👍) to prioritize it.
2. Leave comments to help us solidify the vision.

### PR DESCRIPTION
**Pull request by ro100e**
## 🛠️ Description

Fixes #863 

### 🔬 How did you test this change?

> - [ ] 🤏 Lint tests
> - [ ] 🤞 PS -WhatIf / az validate
> - [x] 👍 Manually deployed + verified
> - [ ] 💪 Unit tests
> - [ ] 🙌 Integration tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [x] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [ ] ✅ Updated changelog (required for `dev` PRs)
> - [ ] ➡️ Will add log in a future PR (feature branch PRs only)
> - [x] ❎ Log not needed (small/internal change)

### 📖 Did you update documentation?

> - [x] ✅ Public docs in `docs` (required for `dev`)
> - [ ] ✅ Internal dev docs in `src` (required for `dev`)
> - [ ] ➡️ Will add docs in a future PR (feature branch PRs only)
> - [ ] ❎ Docs not needed (small/internal change)

### CODE REVIEW

**Review by arthurclares — APPROVED** [review]

File: docs/_reporting/hubs/configure-scopes.md [hunk 1]
```diff
@@ -201,15 +201,8 @@ If you cannot grant permissions for your scope, you can create Cost Management e
...
-   - **Directory** = (Use the resource ID of the scope<sup>5</sup> you're exporting without the first "/")
-     - _**Billing account:** `providers/Microsoft.Billing/billingAccounts/{billingAccountId}`_
-     - _**Billing profile:** `providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}`_
-     - _**Department:** `providers/Microsoft.Billing/billingAccounts/{billingAccountId}/departments/{departmentId}`_
-
-    <blockquote class="tip" markdown="1">
-      _If you get an error about invalid characters for a billing scope, replace the unsupported characters (e.g., `:`) in the billing account ID with a dash (`-`)._
-    </blockquote>
-
+   - **Directory** = (Specify the storage container and directory path<sup>5</sup> for the export file)
```
Comment (flanakin) [inline docs/_reporting/hubs/configure-scopes.md:204]: The container shouldn't be in the path and we should stress that it be unique. There could be data ingestion issues if they use the same path for 2 exports. I'd rather keep the suggestions, but we can simplify them. We can probably not mention the MCA billing account ID, which is the only problem.
```suggestion
   - **Directory** = (Specify a unique path for this scope<sup>5</sup>)
     - _**EA billing account:** `billingAccounts/{enrollment-number}`_
     - _**MCA billing profile:** `billingProfiles/{billing-profile-id}`_
     - _**Subscription:** `subscriptions/{subscription-id}`_
     - _**Resource group:** `subscriptions/{subscription-id}/resourceGroups/{rg-name}`_
```

File: docs/_reporting/hubs/configure-scopes.md [hunk 2]
```diff
@@ -222,7 +215,7 @@ _<sup>1) FinOps hubs 0.2 and beyond requires FOCUS cost data. As of July 2024, t
...
-_<sup>5) A "scope" is an Azure construct that contains resources or enables purchasing services, like a resource group, subscription, management group, or billing account. The resource ID for a scope will be the Azure Resource Manager URI that identifies the scope (e.g., "/subscriptions/###" for a subscription or "/providers/Microsoft.Billing/billingAccounts/###" for a billing account). To learn more, see [Understand and work with scopes](https://aka.ms/costmgmt/scopes).</sup>_
+_<sup>5) Name the storage directory after an identifiable name or ID to assist when troubleshooting through cost exports. This can be the subscription name or subscription ID for a subscription scope (e.g., "corp-test-001" or "########-####-####-####-############"). This can also be the billing account name or the billing account ID  for a billing account scope (e.g., "billing name" or "#######"). To learn more, see [Understand and work with scopes](https://aka.ms/costmgmt/scopes).</sup>_
```
Comment (flanakin) [inline docs/_reporting/hubs/configure-scopes.md:218]: If we keep examples above, we can simplify this and probably just focus on the uniqueness issue.
```suggestion
_<sup>5) Export paths can be any value but must be unique per scope. We recommended using a path that identifies the source scope (e.g., subscription or billing account). If 2 scopes share the same path, there could be ingestion errors.</sup>_
```

**Review by flanakin — APPROVED** [review]

File: docs/_reporting/hubs/configure-scopes.md [hunk 3]
```diff
@@ -36,43 +36,6 @@ For the most seamless experience, we recommend allowing FinOps hubs to manage ex
 
 <br>
 
-
-## 🛠️ Configure exports manually
```
Comment (flanakin) [inline docs/_reporting/hubs/configure-scopes.md:40]: Did you mean to get rid of this? Don't we need it?

**Review by flanakin — CHANGES_REQUESTED** [review]

**Review by arthurclares — CHANGES_REQUESTED** [review]
The manual exports should be the first option. Please, revert it back.

**Review by arthurclares — APPROVED** [review]

File: docs/_reporting/hubs/configure-scopes.md (hunk 3 above)
Comment (ro100e) [reply docs/_reporting/hubs/configure-scopes.md:40]: yes, I was getting a merge conflict

**Review by ro100e — COMMENTED** [review]
Ok

**Review by flanakin — APPROVED** [review]

**Pull request merged**